package.json

```diff
@@ -50,7 +53,7 @@
     "lint:css": "stylelint 'public/*.css'",
     "lint:js": "eslint .",
     "start": "node server/portal_server",
-    "test": "mocha",
+    "test": "mocha test/integration && mocha test/unit",
```

Comment:
Also, you may be able to reduce this to `"test": "mocha 'test/**/*.test.js'",`, if you don't want to specify each suite separately.